src/components/vote/Vote.jsx

```diff
@@ -0,0 +1,48 @@
+import axios from "axios";
+import React, { useState } from "react";
+import "./vote.css";
+
+const Vote = ({ article }) => {
+  const [toggle, setToggle] = useState(false);
+  const [votes, setVotes] = useState(article.votes);
+
+  const handleClick = () => {
+    setToggle(!toggle);
+    let voteValue = 0;
+    if (toggle) {
+      voteValue = -1;
+    } else {
+      voteValue = 1;
+    }
+    axios
+      .patch(
+        `https://hstestncnews.herokuapp.com/api/articles/${article.article_id}`,
+        { votes: voteValue }
+      )
+      .then(({ data }) => {
+        setVotes(votes + voteValue);
+      });
```

Comment:
You need to handle the error here - what if something goes wrong with the API call? I would undo your toggle, and render and error message. It's not likely to happen, but it's good practice to account for it in your code.